api/internal/utils/utils.go

```diff
@@ -94,3 +95,38 @@ func InterfaceToString(val interface{}) string {
 	str := fmt.Sprintf("%v", val)
 	return str
 }
+
+func GenLabelMap(label string) map[string]string {
+	mp := make(map[string]string)
+
+	if label == "" {
+		return mp
+	}
+
+	labels := strings.Split(label, ",")
+	for _, l := range labels {
+		kv := strings.Split(l, ":")
+		if len(kv) == 2 {
+			mp[kv[0]] = kv[1]
+		} else if len(kv) == 1 {
+			mp[kv[0]] = ""
+		}
```

Comment:
Log the malformed label, it's useful when you debug problems.

Reply:
No, I think it's not a malformed label.It's a way only filter labels by key.

Reply:
I mean the `else` case.

Reply:
Fixed.